src/Tooling/Bit.Tooling.Bswup/README.md

```diff
@@ -2,53 +2,50 @@
 
 to use BitBswup, please follow these steps:
 
-1.Install the `Bit.Tooling.Bswup` nuget package
+1. Install the `Bit.Tooling.Bswup` nuget package
 2. Disable static file caching.You can follow below code in `Startup.cs` file
 ```csharp
 
-public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+app.UseStaticFiles(new StaticFileOptions
 {
-    app.UseStaticFiles(new StaticFileOptions
+    OnPrepareResponse = ctx =>
     {
-        OnPrepareResponse = ctx =>
+        ctx.Context.Response.GetTypedHeaders().CacheControl = new CacheControlHeaderValue()
         {
-            ctx.Context.Response.GetTypedHeaders().CacheControl = new CacheControlHeaderValue()
-            {
-                NoCache = true,
-                NoStore = true
-            };
-        }
-    });
-}
+            NoCache = true,
+            NoStore = true
+        };
+    }
+});
 
 ```
-**In the default document based:**(`index.html` or `_Host.cshtml`)
 
-4.Add an `autostart = "false"` attribute and value to the <script> tag for the Blazor script.
+4. In the default document (`index.html` or `_Host.cshtml`), add an `autostart = "false"` attribute and value to the <script> tag for the Blazor script.
 
 ```html
 
 <script src="_framework/blazor.webassembly.js" autostart="false"></script>
 
 ```
 
-5. Add the `Bit.Tooling.Bswup` reference after the <script> tag for the Blazor script.
+5. In the default document (`index.html` or `_Host.cshtml`), add the `Bit.Tooling.Bswup` reference after the <script> tag for the Blazor script.
```

Comment:
_Host.cshtml, index.html or _Layout.cshtml
This is also wrong in bit component getting started where we add style & script as described in getting started. @maryamhdr